pkg/entity/datapolicy/datapolicy.go

```diff
@@ -120,23 +144,20 @@ func readPolicy(filename string) *DataPolicy {
 	return dataPolicy
 }
 
-func TableIdsCompletion(cmd *cobra.Command, args []string, _ string) ([]string, cobra.ShellCompDirective) {
-	if len(args) != 0 {
-		return nil, cobra.ShellCompDirectiveNoFileComp
-	}
-
+func TableOrDataPolicyIdsCompletion(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 	flags := cmd.Flags()
 
-	platformId, catalogId, blueprint := isBlueprint(flags)
+	blueprint := GetBoolAndErr(flags, common.BlueprintFlag)
+	log.Debugln(fmt.Sprintf("TableOrDataPolicyIdsCompletion blueprint: %v", blueprint))
```

Comment:
think this needs to be removed?

Reply:
Good catch. It doesn't really matter, but we had some issues with completions and therefore we added the debug statements. Will remove!.